config/file-dependency-updater/index.js

```diff
@@ -31,6 +31,27 @@ function removeFile(targetFilePath) {
     })
 }
 
+function isOSX() {
+    return process.platform === 'darwin';
+}
+
+function getOptions() {
+    const options = {
+        ignored: /(^|[\/\\])\../,
+        alwaysStat: true,
+        ignoreInitial: true
+    };
+    if (!isOSX()) {
```

Comment:
@simark We don't do it because it works without it as expected. My guess that increasing the threshold would help, in the doc, they say to get a deterministic behavior one should set it is quite high:
> The appropriate duration setting is heavily dependent on the OS and hardware. For accurate detection this parameter should be relatively high, making file watching much less responsive. Use with caution.